socorro/processor/mozilla_transform_rules.py

```diff
@@ -600,26 +582,9 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
 
 
 class BetaVersionRule(Rule):
-    required_config = Namespace()
-    required_config.add_option(
-        'database_class',
-        doc="the class of the database",
-        default='socorro.external.postgresql.connection_context.'
-                'ConnectionContext',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
-    required_config.add_option(
-        'transaction_executor_class',
-        default="socorro.database.transaction_executor."
-                "TransactionExecutorWithInfiniteBackoff",
-        doc='a class that will manage transactions',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
-
```

Comment:
These two values come from Processor2015 config, so this is dead code. It's very confusing since it seems like it comes from here, but it doesn't anymore.